sat/cli/bootsys/mgmt_power.py

```diff
@@ -260,6 +260,86 @@ def finish_shutdown(hosts, username, password, ncn_shutdown_timeout, ipmi_timeou
             sys.exit(0)
 
 
+def get_disk_entries(ssh_client, ncn):
+    """
+    Retrieve the current boot order from a node via SSH and identify disk entries.
+
+    Args:
+        ssh_client (paramiko.SSHClient): The SSH client connected to the node.
+        ncn (str): The node associated with the ssh_client.
+
+    Returns:
+        str: The boot number of the first disk entry found.
+
+    Raises:
+        SystemExit: If the command fails, an error is logged and the function exits.
+    """
+    command = "efibootmgr"
+    _, stdout, stderr = ssh_client.exec_command(command)
+    error = stderr.read().decode()
+    if error:
+        LOGGER.error(f'Failed to get current boot order for {ncn}: {error}')
+        raise SystemExit(1)
+
+    boot_order_output = stdout.read().decode()
+    lines = boot_order_output.split('\n')
+    for line in lines:
+        if line.startswith("Boot"):
+            parts = line.split()
+            boot_num = parts[0][4:8]
+            boot_desc = " ".join(parts[1:])
```

Comment:
I'd suggest using `maxsplit=1` here, so that you can then avoid having to rejoin everything back together on line 290.

Reply:
I'm going to use regex which is more convenient as per our last discussion